### PR TITLE
OWC 2016 Article 

### PR DESCRIPTION
**Pull request by oralekin**
# WIP

Converted from the original wiki article.
------------------

[old article](https://osu.ppy.sh/wiki/Osu!_World_Cup_2016)

I'm thinking of making a spreadsheet of the groups and not bothering with the tabling

### CODE REVIEW

**Comment by RaikireHiuduo** [comment]
[Organisation.txt](https://github.com/ppy/osu-wiki/files/994848/Organisation.txt)

A direct cp to GitHub annoys me (there is newline spaces, every line).

**Comment by TPGPL** [comment]
http://puu.sh/vOALj/6f5a6243bc.txt fixed Groups

File: wiki/Tournaments/OWC/2016/en.md [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+## Links
+
+- [Livestream](http://www.twitch.tv/osulive)
+- [Statistics](https://docs.google.com/spreadsheets/u/1/d/1rvVSjmdHIb-h8pekmVfIIPDPivrD5fACjNRYGo1U9to/pubhtml)
+- [Discussion Thread](https://osu.ppy.sh/forum/t/507270)
+- [Stage Recaps](https://www.youtube.com/playlist?list=PLqJuZKl72XH_El27Es8RsxW5NXDUVeRRl) (by Feurigel)
+
+-----------
+
+![Podium](podium.png)
```
Comment (MegaApplePi) [inline wiki/Tournaments/OWC/2016/en.md]: wrong file extension, the file format for this file is in `.jpg`

File: wiki/Tournaments/OWC/2016/en.md [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+
+-----------
+
+![Brackets](brackets.png)
```
Comment (MegaApplePi) [inline wiki/Tournaments/OWC/2016/en.md]: ditto; wrong file extension, the file format for this file is in `.jpg`

**Pull request closed without merging**